### DISCUSSION TITLE
Testing antd popover.

### DISCUSSION THREAD

**Discussion by dvkruchinin** [cypress-io/cypress#14321] [Questions and Help]
Hello.

When updating the ant version to 4x, I encountered a problem testing popover.
At transition popover, the css "popover-events: none" appears. When using the Firefox browser in headless mode, the error of getting a popover is common. This error also occurs in the Chrome browser, but much less often. Without the headless mod in the browser console, the message ```But it has css:          pointer-events: none``` is visible when trying to click on popover.

Sample test:
```
describe('Test for cypress issue. ANTD popover.', () => {
    it('Visit to andt', () => {
        cy.visit('https://ant.design/components/popover/');
        cy.contains('Hover me').trigger('mouseover');
        cy.get('.ant-popover').should('be.visible').click();
    });
});
```

In our app there are also other elements on popover that need to interact with. An error occurs
```
CypressError: Timed out retrying: `cy.click()` failed because this element: <element inside the popover>
has CSS `pointer-events: none`, inherited from this element: .ant-popover
```
Tried different combinations of using ```should```, ```expect```. For example:
```
cy.get('.ant-popover')
        .should('not.have.class', 'ant-popover-hidden')
        .and('not.have.class', 'zoom-big-enter')
        .and('not.have.css', 'pointer-events', 'none')
```
Please help how to achieve success in this case. ```cy. wait()``` I do not want to use because waiting can take different time.

**Comment by cd-a** [comment]
@dvkruchinin Did you find a fix for this?
